deployment/sandbox-v2/helm/charts/regproc/values.template.j2

```diff
@@ -294,11 +360,11 @@ services:
       hazelcastPort: 5701
       containerPort: 8099
       servicePort: 80
-    ingressPath: '' 
+    ingressPath: ''
     healthUrl: /registrationprocessor/v1/print-stage/actuator/health
 
   trans:
-    serviceName: {{ versions.services.regproc.trans.keys()[0] }} 
```

Comment:
If possible avoid these space changes, so git does not show that line is changed